src/test/java/fi/metatavu/kuntaapi/test/json/JSONAssertCustomizations.java

```diff
@@ -0,0 +1,44 @@
+package fi.metatavu.kuntaapi.test.json;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import org.skyscreamer.jsonassert.Customization;
+
+/**
+ * JSONAssert customization utils
+ * 
+ * @author Antti Leppä
+ */
+public class JSONAssertCustomizations {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)